### ISSUE TITLE
FAQ entry about referencing local script: "including an input of `type: File` in the script itself"?

### ISSUE THREAD

**Issue by zoe-translates** [common-workflow-language/user_guide#385]
Am I correct in understanding that the intention here was "including *the script as* an input of `type:File` in *the CWL document* itself"?

https://github.com/common-workflow-language/user_guide/blob/dd897f448ef3db0d4528c4e67b387a2cc04752a1/src/faq.md?plain=1#L115-L132

In the context of this entry, "the script" refers to the executable script the user wants to reference in the CWL file. I think the code describes how to make that script "an input of `type:File`", rather than how to put some CWL-specific data ("input of `type:File`") in that executable script.

**Comment by mr-c** [comment]
> I think the code describes how to make that script "an input of `type:File`", rather than how to put some CWL-specific data ("input of `type:File`") in that executable script.

This is correct. Literally embedding a script is also possible, but only recommended for short scripts (maybe one screen of text).
